content/docs/Configuration/_index.md

```diff
@@ -789,6 +789,13 @@ Requirements:
   in dist-git should be set to `Monitoring`).
 * [`upstream_project_url`](#upstream_project_url) is defined in the configuration.
 
+{{< hint info >}}
+Upstreams (defined in `upstream_project_url`) hosted in these Git forges are currently supported: https://github.com, https://gitlab.com, https://gitlab.freedesktop.org,
+https://gitlab.gnome.org, https://salsa.debian.org.
```

Comment:
> https://salsa.debian.org

@lachmanfrantisek is this right or packit/ogr#771 needs to be in prod so that this works?

Reply:
Missed it... Sorry. It's required, but It's already been on prod since yesterday. Sadly, it requires also https://github.com/packit/ogr/pull/775 to support SRPM builds in Copr.

I've added the instances also to the guide: #613

Reply:
thanks! So for the purposes of `pull-from-upstream`, this should be now usable, right?